bench/dune

```diff
@@ -57,3 +57,22 @@
  (name lzld)
  (modules lzld)
  (libraries decompress.de lz_landmarks))
+
+(library
+ (name de_landmarks)
+ (optional)
+ (modules de_landmarks)
```

Comment:
You can add `(flags (:standard -w -55))` to keep some function with the `inline` tag (on the `de.ml`) and be able to compile the landmarks target without trouble.

Reply:
Thanks a lot, it's pushed